apps/docs/components/Utils/ThemeProvider/ThemeHelper.tsx

```diff
@@ -0,0 +1,16 @@
+export type ColorScheme = "light" | "dark";
```

Comment:
rename this file to get-initial-color-mode.ts or inline it in the Theme Provider

Reply:
file extension should not be tsx if there is no JSX in it (the <Div> like tags)

Reply:
inline it in the Theme Provider would create SSR issues

Reply:
> file extension should not be tsx if there is no JSX in it (the

Since it's imported by tsx files they seem to require typing

Reply:
moving the getInitialColorMode inside the ThemePRoviderFile will not cause SSR issues.

For the extension, i meant the file extension should be changed to ThemeHelper.ts. It will not cause issues

Reply:
> For the extension, i meant the file extension should be changed to ThemeHelper.ts. It will not cause issues
Yes I realized afterward that the x was not necessary :)